### PR TITLE
NAS-101830 Update helptext in Network > Interfaces > ADD.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 1]
```diff
@@ -4,7 +4,11 @@ import { rangeValidator } from '../../../pages/common/entity/entity-form/validat
 
 export default {
 int_type_placeholder: T('Type'),
-int_type_tooltip: T('Choose the type of interface. <i>Bridge</i> creates a logical link between multiple networks. Link Aggregation combines multiple network connections into a single interface. A virtual LAN (VLAN) partitions and isolates a segment of the connection.'),
+int_type_tooltip: T('Read-only when editing an interface. Choose the\
+ type of interface. <i>Bridge</i> creates a logical link between\
+ multiple networks. Link Aggregation combines multiple network\
+ connections into a single interface. A virtual LAN (VLAN) partitions\
```
Comment (Mrt134) [inline src/app/helptext/network/interfaces/interfaces-form.ts]:
```suggestion
 connections into a single interface. A Virtual LAN (VLAN) partitions\
```
match acronym

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 2]
```diff
@@ -20,7 +24,9 @@ int_description_placeholder : T('Description'),
 int_description_tooltip : T('Enter a description of the interface.'),
 
 int_name_placeholder : T('Name'),
-int_name_tooltip : T('Enter a name to use for the the interface. Use the format laggX, vlanX, or bridgeX where X is a number representing a non-parent interface.'),
+int_name_tooltip : T('Read-only when editing an interface. Enter a\
+ name to use for the the interface. Use the format laggX, vlanX, or\
```
Comment (Mrt134) [inline src/app/helptext/network/interfaces/interfaces-form.ts]: Does "X" in the examples need to be italic?

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 3]
```diff
@@ -4,7 +4,11 @@ import { rangeValidator } from '../../../pages/common/entity/entity-form/validat
 
 export default {
 int_type_placeholder: T('Type'),
-int_type_tooltip: T('Choose the type of interface. <i>Bridge</i> creates a logical link between multiple networks. Link Aggregation combines multiple network connections into a single interface. A virtual LAN (VLAN) partitions and isolates a segment of the connection.'),
+int_type_tooltip: T('Read-only when editing an interface. Choose the\
+ type of interface. <i>Bridge</i> creates a logical link between\
+ multiple networks. Link Aggregation combines multiple network\
```
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]:
```suggestion
 multiple networks. <i>Link Aggregation</i> combines multiple network\
```

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 4]
```diff
@@ -4,7 +4,11 @@ import { rangeValidator } from '../../../pages/common/entity/entity-form/validat
 
 export default {
 int_type_placeholder: T('Type'),
-int_type_tooltip: T('Choose the type of interface. <i>Bridge</i> creates a logical link between multiple networks. Link Aggregation combines multiple network connections into a single interface. A virtual LAN (VLAN) partitions and isolates a segment of the connection.'),
+int_type_tooltip: T('Read-only when editing an interface. Choose the\
```
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]: Tell what it is first, then other details like when it is read-only.

File: src/app/helptext/network/interfaces/interfaces-form.ts (hunk 1 above)
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]:
```suggestion
 connections into a single interface. A <i>Virtual LAN</i> (VLAN) partitions\
```

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 5]
```diff
@@ -20,7 +24,9 @@ int_description_placeholder : T('Description'),
 int_description_tooltip : T('Enter a description of the interface.'),
 
 int_name_placeholder : T('Name'),
-int_name_tooltip : T('Enter a name to use for the the interface. Use the format laggX, vlanX, or bridgeX where X is a number representing a non-parent interface.'),
+int_name_tooltip : T('Read-only when editing an interface. Enter a\
```
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]: Again, description of what it is first, then other details like when it is read-only.

File: src/app/helptext/network/interfaces/interfaces-form.ts (hunk 2 above)
Comment (wblock) [reply src/app/helptext/network/interfaces/interfaces-form.ts]: Well, yes, but the interface names should also be in a computer font.  Can we use the <samp> tag?
```suggestion
 name to use for the the interface. Use the format <samp>lagg<i>X</i></samp>, <samp>vlan<i>X</i></samp>, or\
```

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 6]
```diff
@@ -20,7 +24,9 @@ int_description_placeholder : T('Description'),
...
+ name to use for the the interface. Use the format laggX, vlanX, or\
+ bridgeX where X is a number representing a non-parent interface.'),
```
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]:
```suggestion
 <samp>bridge<i>X</i></samp> where <i>X</i> is a number representing a non-parent interface.'),
```
"Non-parent interface" is not going to have much meaning to most users.

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 7]
```diff
@@ -4,23 +4,32 @@ import { rangeValidator } from '../../../pages/common/entity/entity-form/validat
 
 export default {
 int_type_placeholder: T('Type'),
-int_type_tooltip: T('Choose the type of interface. <i>Bridge</i> creates a logical link between multiple networks. Link Aggregation combines multiple network connections into a single interface. A virtual LAN (VLAN) partitions and isolates a segment of the connection.'),
+int_type_tooltip: T('Choose the\
+ type of interface. <i>Bridge</i> creates a logical link between\
+ multiple networks. <i>Link Aggregation</i> combines multiple network\
+ connections into a single interface. A Virtual LAN (VLAN) partitions\
```
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]:
```suggestion
 connections into a single interface. A <i>Virtual LAN (VLAN)</i> partitions\
```

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 8]
```diff
@@ -4,23 +4,32 @@ import { rangeValidator } from '../../../pages/common/entity/entity-form/validat
...
+int_interface_tooltip : T('Enter the FreeBSD device name of the\
+ interface. This cannot be changed after the interface is created.'),
 int_interface_validation : [ Validators.required ],
 
 int_description_placeholder : T('Description'),
 int_description_tooltip : T('Enter a description of the interface.'),
 
 int_name_placeholder : T('Name'),
-int_name_tooltip : T('Enter a name to use for the the interface. Use the format laggX, vlanX, or bridgeX where X is a number representing a non-parent interface.'),
+int_name_tooltip : T('Enter a name to use for the the interface.\
```
Comment (wblock) [inline src/app/helptext/network/interfaces/interfaces-form.ts]:
```suggestion
int_name_tooltip : T('Enter a name for the interface.\
```

**Pull request merged**